about_me-style.css

```diff
@@ -0,0 +1,31 @@
+body{
```

Comment:
I mentioned earlier that you ultimately want one master css file that is applied to all of your pages. For example, you probably want the main body font to be the same on every page, so you don't need a different file for "about me" vs. your home page.

But then there might be some styles that are specific to a single page. For those, you're welcome to make a second css file and include both of them, or you can put the page-specific styles in the `<style>` tag in the head of the page.

Let me know if you want to discuss that more.